### PR TITLE
Reminders migration updates

### PR DESCRIPTION
**Pull request by gcapalbo**
Two small updates to the reminders migration process.

@dannyroberts

### CODE REVIEW

File: corehq/apps/reminders/management/commands/migrate_to_new_reminders.py [hunk 1]
```diff
@@ -498,6 +499,11 @@ def handle(self, domain, **options):
         if not check_only:
             self.ensure_migration_flag_enabled(domain)
 
+        if not check_only and couch_sql_migration_in_progress(domain):
```
Comment (gcapalbo) [inline corehq/apps/reminders/management/commands/migrate_to_new_reminders.py]: @jmtroth0 if you wanted to check for the reminders migration in your script as well you could just check [REMINDERS_MIGRATION_IN_PROGRESS.enabled(domain)](https://github.com/dimagi/commcare-hq/blob/af69cf1be9d1d4372b21a9e26467b09545fe45f8/corehq/toggles.py#L1146)

**Comment by gcapalbo** [comment]
:heavy_check_mark:

**Comment by gcapalbo** [comment]
bump @dannyroberts

**Review by dannyroberts — APPROVED** [review]

File: corehq/apps/reminders/management/commands/migrate_to_new_reminders.py (hunk 1 above)
Comment (jmtroth0) [reply corehq/apps/reminders/management/commands/migrate_to_new_reminders.py]: Thanks! https://github.com/dimagi/commcare-hq/pull/20711

**Pull request merged**